.github/workflows/build-dumb-pypi.yml

```
@@ -180,7 +181,8 @@ jobs:
       env:
         _ANSIBLE_SDIST_FROM_MAKEFILE: 1
       run: |
-        make sdist_check
+        packaging/release.py build --allow-dirty
+        packaging/release.py test
         python -m setup sdist --dist-dir=../dist
```

If the version bump and related activities aren't needed, it's fine to just do:
```suggestion
      run: python -m build --config-setting=--build-manpages  --outdir=../dist
```

---

I'm not sure whether it's needed.

---

df06fcdd69d62d1025d2f89f7872f871d1abb76c basically does this.